deployment/windows/jellyfin.nsi

```diff
@@ -0,0 +1,381 @@
+; This can be removed once stable.
+!verbose 4
+;--------------------------------
+!define SF_USELECTED  0 ; used to check selected options status, rest are inherited from Sections.nsh
+
+    !include "MUI2.nsh"
+	!include "Sections.nsh"
+	!include "LogicLib.nsh"
+
+; Global variables that we'll use
+    Var _JELLYFINVERSION_
+    Var _JELLYFINDATADIR_
+	Var _INSTALLSERVICE_
+	Var _SERVICESTART_
+	Var _NETWORKSERVICEACCOUNT_
+	Var _EXISTINGINSTALLATION_
+	Var _EXISTINGSERVICE_
+	Var _CUSTOMDATAFOLDER_
+
+!if ${NSIS_PTR_SIZE} > 4
+  !define BITS 64
+  !define NAMESUFFIX " (64 bit)"
+!else
+  !define BITS 32
+  !define NAMESUFFIX ""
+!endif
+
+;--------------------------------
+
+	!define REG_UNINST_KEY "Software\Microsoft\Windows\CurrentVersion\Uninstall\Jellyfin" ;Registry to show up in Add/Remove Programs
+	
+    !getdllversion "$%InstallLocation%\jellyfin.dll" ver_ ;Align installer version with jellyfin.dll version
+    Name "Jellyfin Server ${ver_1}.${ver_2}.${ver_3}" ; This is referred in various header text labels	
+    OutFile "jellyfin_${ver_1}.${ver_2}.${ver_3}_windows.exe" ; Naming convention jellyfin_{version}_windows-{arch].exe
+    BrandingText "Jellyfin Server ${ver_1}.${ver_2}.${ver_3} Installer" ; This shows in just over the buttons
+
+; installer attributes, these show up in details tab on installer properties
+    VIProductVersion "${ver_1}.${ver_2}.${ver_3}.0" ; VIProductVersion format, should be X.X.X.X
+    VIFileVersion "${ver_1}.${ver_2}.${ver_3}.0" ; VIFileVersion format, should be X.X.X.X
+    VIAddVersionKey "ProductName" "Jellyfin Server"
+    VIAddVersionKey "FileVersion" "${ver_1}.${ver_2}.${ver_3}.0"
+	VIAddVersionKey "LegalCopyright" "Jellyfin, Free Software Media System"
+	VIAddVersionKey "FileDescription" "Jellyfin Server"  
+	
+;TODO, check defaults
+    InstallDir "$PROGRAMFILES\Jellyfin" ;Default installation folder
+    InstallDirRegKey HKLM "Software\Jellyfin" "InstallFolder" ;Read the registry for install folder,
+  
+    RequestExecutionLevel admin ; ask it upfront for service control, and installing in priv folders
+	
+    CRCCheck on ; make sure the installer wasn't corrupted while downloading
+	
+    !define MUI_ABORTWARNING ;Prompts user in case of aborting install
+
+; TODO: Replace with nice Jellyfin Icons
+	!define MUI_ICON "${NSISDIR}\Contrib\Graphics\Icons\nsis3-install.ico" ; Installer Icon
+	!define MUI_UNICON "${NSISDIR}\Contrib\Graphics\Icons\nsis3-uninstall.ico" ; Uninstaller Icon
+	
+	!define MUI_HEADERIMAGE
+	!define MUI_HEADERIMAGE_BITMAP "${NSISDIR}\Contrib\Graphics\Header\nsis3-branding.bmp"
+	!define MUI_WELCOMEFINISHPAGE_BITMAP "${NSISDIR}\Contrib\Graphics\Wizard\nsis3-branding.bmp"
+
+;--------------------------------
+;Pages
+
+; Welcome Page
+	!define MUI_WELCOMEPAGE_TEXT "The installer will ask for details to install Jellyfin Server.$\r$\n$\r$\n$\r$\n\
+	ADVANCED:$\r$\n\
+	The default service install uses Local System account and is sufficient for most users. $\r$\n$\r$\n\
```

Comment:
I still feel we should err on the side of caution with running the service as Local System, it's the equivalent of running jellyfin as root on *nix.

Reply:
Yeah, my intent was to have a default account to use and provide the other one as optional. I've added the code to handle these cases, just looking around a bit more and will update the code based on that.